chat/chat-api/api/src/java/org/sakaiproject/chat2/model/TransferableChatMessage.java

```diff
@@ -0,0 +1,95 @@
+package org.sakaiproject.chat2.model;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Date;
+
+import lombok.Data;
+
+/**
+ * 
+ * Class used to transfer data through jGroups
+ *
+ */
+@Data
+public class TransferableChatMessage implements Serializable {
+
+	public enum MessageType {
+	    CHAT,
+	    HEARTBEAT,
+	    CLEAR,
+	    REMOVE;
+	}
+	
+	public MessageType type;
+	public String id;
+	public String owner;
+	public String siteId;
+	public String channelId;
+	public String content;
+	public long timestamp;
```

Comment:
Correct just use java.util.Date and then convert to java 8 time for now. H5 supports the new java 8 time types.